packages/web-config-server/src/apiV1/measureData.js

```diff
@@ -152,69 +196,13 @@ export default class extends DataAggregatingRouteHandler {
         }
       }),
     );
-
-    // start fetching options
-    const optionsTasks = overlays.map(o => this.fetchMeasureOptions(o, this.query));
     // start fetching actual data
     const shouldFetchSiblings = this.query.shouldShowAllParentCountryResults === 'true';
-    const dataTasks = overlays.map(o => this.fetchMeasureData(o, shouldFetchSiblings));
-
-    // wait for fetches to complete
-    const measureOptions = await Promise.all(optionsTasks);
-    const measureDataResponsesByMeasureId = (
-      await Promise.all(dataTasks)
-    ).reduce((dataResponse, current) => ({ ...dataResponse, ...current }));
-
```

Comment:
Group these code as function `fetchMeasureOptions`, and rename `fetchMeasureOptions` to `fetchMeasureOption`